### PR TITLE
fix: postcss-loader config not support deepfreezed

### PR DESCRIPTION
**Pull request by 9aoy**
# PR Details

postcss-loader config not support deepfreezed

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/builder/builder-webpack-provider/src/plugins/css.ts [hunk 1]
```diff
@@ -127,7 +128,8 @@ export async function applyBaseCSSRule(
       mergedConfig.postcssOptions.plugins!.push(...extraPlugins);
     }
 
-    return mergedConfig as ProcessOptions & {
+    // postcss-loader will modify config
+    return _.cloneDeep(mergedConfig) as ProcessOptions & {
```
Comment (chenjiahan) [inline packages/builder/builder-webpack-provider/src/plugins/css.ts:132]: postcss 插件应该是比较复杂的对象，这里深拷贝是不是有一些性能开销？ 可以看下耗时

Comment (9aoy) [reply packages/builder/builder-webpack-provider/src/plugins/css.ts:132]: 拿默认配置测了下，时间可以忽略不计。看了下很多 plugin 都是 function，这部分不会执行深拷贝

image

Comment (chenjiahan) [reply packages/builder/builder-webpack-provider/src/plugins/css.ts:132]: 是不是 deepClone `config.tools.postcss` 这个值就可以，不需要 deepClonse 合并后的对象

Comment (9aoy) [reply packages/builder/builder-webpack-provider/src/plugins/css.ts]: 机智！

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**